rasa/nlu/extractors/extractor.py

```diff
@@ -333,3 +346,149 @@ def filter_trainable_entities(
             )
 
         return filtered
+
+    def create_entities(
```

Comment:
I don't think this name the best describes what it does

Reply:
any suggestion? `create_entities_from_predictions`?

Reply:
isn't it rather converting predictions into entities? I think `create` here is strange, because at least I associate "creating" differently